ibtk/src/boundary/cf_interface/CartCellDoubleQuadraticCFInterpolation.cpp

```diff
@@ -481,13 +465,13 @@ CartCellDoubleQuadraticCFInterpolation::postprocessRefine_expensive(Patch<NDIM>&
 
         const Box<NDIM>& patch_box_fine = fine.getBox();
         const Index<NDIM>& patch_lower_fine = patch_box_fine.lower();
-        Pointer<CartesianPatchGeometry<NDIM> > pgeom_fine = fine.getPatchGeometry();
+        Pointer<CartesianPatchGeometry<NDIM>> pgeom_fine = fine.getPatchGeometry();
```

Comment:
But we cannot use it here without changing the rhs, because we are relying on `Pointer`'s ability to automatically downcast from a base class to a derived class.